builds.expected

```diff
@@ -1916,6 +1916,7 @@ ocurrent/opam-staging:archlinux-opam-amd64 -> ocaml/opam:archlinux-opam
 	# syntax=docker/dockerfile:1
 
 	FROM ocurrent/opam-staging:archlinux-opam-amd64
+	RUN opam repo add ocaml-patches-overlay git+https://github.com/mtelvers/opam-repository#patches --set-default
```

Comment:
Should we put this in a fork under the ocurrent organization instead? I regret the bureaucratic request, but it seems like a bit better practice than using our personal accounts for this kind of thing.